locale/cs/LC_MESSAGES/django.po

```diff
@@ -3206,7 +3206,7 @@ msgid ""
 "Our volunteers haven't translated this article into <bdi>%(locale)s</bdi> "
 "yet. <a href=\"%(help_link)s\">Join us and help get the job done!</a>"
 msgstr ""
-"Naši dobrovolníci ještě tento článek do jazyka <bdi>%s (locale)</bdi> "
+"Naši dobrovolníci ještě tento článek do jazyka <bdi>%(locale)s</bdi> "
```

Comment:
Thank you for letting me know. The malformed format string is even my fault, I am very sorry for that. I have fixed it in Pontoon.

Btw. great to see the new Pontoon direct links in action. ;)